dspace-api/src/test/java/org/dspace/app/packager/PackagerIT.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.packager;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.UUID;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import org.dspace.AbstractIntegrationTestWithDatabase;
+import org.dspace.builder.CollectionBuilder;
+import org.dspace.builder.CommunityBuilder;
+import org.dspace.builder.ItemBuilder;
+import org.dspace.content.Collection;
+import org.dspace.content.Community;
+import org.dspace.content.Item;
+import org.dspace.content.crosswalk.MetadataValidationException;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.packager.METSManifest;
+import org.dspace.content.service.ItemService;
+import org.jdom.Element;
+import org.junit.Test;
+
+public class PackagerIT extends AbstractIntegrationTestWithDatabase {
+
+    private ItemService itemService = ContentServiceFactory.getInstance().getItemService();
+    protected Community child1;
+    protected Collection col1;
+    protected Item article;
+    File tempFile;
+
+    @Test
+    public void packagerExportUUIDTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        createTemplate();
+        createFiles();
+        try {
+            performExportScript(article.getHandle(), tempFile);
+            assertTrue(tempFile.length() > 0);
+            String idStr = getID();
+            assertEquals(idStr, article.getID().toString());
+        } finally {
+            tempFile.delete();
+        }
+    }
+
+    @Test
+    public void packagerImportUUIDTest() throws Exception {
```

Comment:
This method appears to only test if Item UUIDs are restored.  We should have similar tests for Community & Collection to verify they work similarly.

As a sidenote...you may have seen it, but we already have a `ITDSpaceAIP` class which does a very thorough export/import of a hierarchy of AIPs.  If you find it easier, you could simply update that IT to also verify that UUIDs are restored for various object types.  It's up to you though, as having a separate IT class is reasonable as well.

Reply:
Left my changed in its own class because I extend AbstractIntegrationTestWithDatabase which allows me to call the runDspaceScript()